src/main/java/domain/WinningLottery.java

```diff
@@ -0,0 +1,60 @@
+package domain;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class WinningLottery {
+    private static final String LOTTERY_NUMBER_LACK_ERROR_MESSAGE = "error : 로또 번호는 중복되지 않는 번호 6개 입니다.";
+    private static final int LOTTERY_SIZE_MAX = 6;
+    private static final boolean MATCH_NUMBER_TRUE = true;
+    private static final boolean MATCH_NUMBER_FALSE = false;
+
+    private final Set<LotteryNumber> winningLottery;
+    private final int bonusBall;
+
+    public WinningLottery(Set<Integer> numbers, int bonusBall) {
+        this.winningLottery = changeToLotteryNumber(numbers);
+        this.bonusBall = bonusBall;
+    }
+
+    public Set<LotteryNumber> changeToLotteryNumber(Set<Integer> numbers) {
+        validWinningLottery(numbers.size());
+
+        return of(numbers);
+    }
+
+    private void validWinningLottery(int count) {
+        if (count < LOTTERY_SIZE_MAX) {
+            throw new IllegalArgumentException(LOTTERY_NUMBER_LACK_ERROR_MESSAGE);
+        }
+    }
+
+    private Set<LotteryNumber> of(Set<Integer> numbers) {
+        return numbers.stream()
+                .map(LotteryNumber::new)
+                .collect(Collectors.toSet());
+    }
+
+    public boolean matchLottery(int lotteryNumber) {
```

Comment:
정의한 불변 객체를 그대로 사용한다면 contains와 같은 메소드로 확인할수 있지 않을까요 ㅎㅎ
```suggestion
    public boolean matchLottery(LotteryNumber lotteryNumber) {
```